e2e/test/scenarios/admin/settings/email.cy.spec.js

```diff
@@ -102,36 +125,11 @@ describe("scenarios > admin > settings > email settings", () => {
       // Make sure some settings are already there
       setupSMTP();
 
-      cy.visit("/admin/settings/email");
+      cy.visit("/admin/settings/email/smtp");
       // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
       cy.findByText("Send test email").scrollIntoView();
       // Needed to scroll the page down first to be able to use findByRole() - it fails otherwise
       cy.button("Save changes").should("be.disabled");
     },
   );
-
-  it("should not reset previously populated fields when validation fails for just one of them (metabase#16226)", () => {
```

Comment:
This test was deleted since the additional email settings are now isolated from the SMTP configuration. They will no longer have an effect on one another.